doc/source/getting_started/develop_pymapdl.rst

```diff
@@ -0,0 +1,337 @@
+details how merging a pull request would impact coverage. It is one of the
+checks that must run successfully to merge code changes.
+
+
+.. figure:: ../images/codecov_increase.png
+    :width: 400pt
+
+
+Coverage example
+----------------
+
+To show how the coverage works, assume that you have
+this library:
+
+**Awesome library**
+
+
+.. code:: python
+
+    def get_report_colors(theme):
+        if theme == "weather":
+            colors = ["blue", "lightblue", "grey"]
+        elif theme == "traffic":
+            colors = ["red", "orange", "yellow"]
+        else:
+            colors = ["red", "blue", "green"]
+
+        return colors
+
+
+**Tests**
+
+You can opt to run the tests with this configuration:
+
+.. code:: python
+
+   def test_get_report_colors():
+       assert get_report_colors("weather") == ["blue", "lightblue", "grey"]
+       assert get_report_colors("traffic") == ["red", "orange", "yellow"]
+       assert get_report_colors("other") == ["red", "blue", "green"]
+
+
+Or, if a method is a bit more complex, you can split the case in different tests:
+
+.. code:: python
+
+    def test_get_report_colors_weather():
+        assert get_report_colors("weather") == ["blue", "lightblue", "grey"]
+
+
+    def test_get_report_colors_traffic():
+        assert get_report_colors("traffic") == ["red", "orange", "yellow"]
+
+
+    def test_get_report_colors_other():
+        assert get_report_colors("other") == ["red", "blue", "green"]
+
+
+While the code coverage in either case is 100% for the function, the second case is
+more useful for debugging the function.
+
+
+Continuous Integration and Continuous Deployment (CI/CD) approach
+-----------------------------------------------------------------
+
+Unit tests and integration tests are part of Continuous Integration (CI). 
+The automation of testing, monitoring, and deployment of newly added
+code allows Continuous Deployment (CD) throughout the app lifecycle,
+providing a comprehensive CI/CD approach.
+
+.. figure:: ../images/cicd.jpg
+    :width: 300pt
+
+Creation of a unit test
+-----------------------
+
+In the PyMAPDL repository, `pytest <pytest_>`_ is used to run tests. 
+
+The name of a ``pytest`` file must be in the form ``test_XXX.py``, where ``XXX``
+is either the function, method, or class that you are testing or some other explicative
+name. In the command line, the ``-k`` argument can be used to filter the tests to run.
+For more information, see `pytest usage <pytest_usage_>`_.
+
+Here are some guidelines for creating good unit tests: 
+
+- Assign long and descriptive names to tests.
+- Use the `Codecov <codecov_>`_ tool to ensure all implemented code is tested.
+- Check that tests return the same results each time. 
+- Verify that tests are independent.
+- Write tests that verify only one part of the code at a time.
+- Make tests as short and fast as possible.
+
+`What makes a good unit test <article_good_unit_test_>`_ 
```

Comment:
Should this title be a question?

Reply:
I agree on the limitation of `links.rst`. Maybe we could work on an extension for VSCode. It should be easy to do :)